### DISCUSSION TITLE
How can I use docker layer cache with RunnerSet?

### DISCUSSION THREAD

**Discussion by JaeHyeokLee** [actions/actions-runner-controller#788] [Questions] [answered]
Hi, I made docker build action using action runner controller.
But docker image build action  is too slow because [PR](https://github.com/actions-runner-controller/actions-runner-controller/issues/612) (volume claim for Runnerset)  is not merged.
Then, how can I use docker layer cache without volume claim? Is it possible to use docker layer cache with RunnerSet (or RunnerDeployment) with ephemeral: false? Or is there any solution for faster docker build?

**Comment by toast-gear** [comment, marked as answer]
https://github.com/actions-runner-controller/actions-runner-controller/discussions/711#discussioncomment-1128825 see the last bullet point for one way for caching docker layers without stateful runners
